loops-and-arrays.js

```diff
@@ -1,11 +1,11 @@
-// Opdracht 1
+// Opdracht 1 // 
```

Comment:
In de opdracht staat dat je ook de namen voor de +je moet loggen. Maar dat stelt natuurlijk verder  niets voor.